### DISCUSSION TITLE
Is there a way to bind an "IsValid" value from a ValidationsRef to enable/disable Buttons

### DISCUSSION THREAD

**Discussion by TorreyGarland** [Megabit/Blazorise#3861] [Q&A]
What I am trying to do is bind a value to from a ValidationsRef to enable or disable a button inside of a form based on whether or not the form is currently valid.

right now, using a reactiveui viewmodel with ICommands and Blazorise fluentvalidation.  

I can't find any public property that would indicate if the form is currently valid or not.

**Comment by stsrki** [comment]
You can bind the `StatusChanged` event on Validations. And the check for `e.Success` parameter.

**Comment by TorreyGarland** [comment]
Yeah, I tried this, but it does not work as expected.  

I am using Blazorise.FluentValidation and I think it does not work as expected because of how either Blazorise.FV or FV validates properties individually.  I am not sure if it is working on the viewmodel being validated as a whole.

**Comment by TorreyGarland** [comment]
Actually, you were right and this works, with a little extra code.

The key was using bool? instead of bool for "IsValid".
